atlp-rwanda/cabal-phantom-backend
# #KtzOqedR CRUD Operations for routes

### **What about this PR?**
This PR has CRUD Operations for routes

### **Description of Task to be completed**
It should have the following endpoint:
- GET /api/v1/routes

- GET /api/v1/routes?page={page}&limit={limit}

- POST /api/v1/routes

- GET /api/v1/routes/id

- PATCH /api/v1/routes/id

- DELETE /api/v1/routes/id

### **How should be tested?**
- You should use [swagger documentation](https://phantom-caba-ft-crud-fo-fjhnuy.herokuapp.com/api-docs/) and login with the credential below

- You should also use postman with this link https://phantom-caba-ft-crud-fo-fjhnuy.herokuapp.com/

- Copy token and paste it in documentation (authorization) or in header if you are using postman.

- Test all endpoints specified above

Use these credentials for login: `{  "email": "admin@phantom.com", "password": "admin" }`

### **Relevant pivots tracker**
[#KtzOqedR](https://trello.com/c/KtzOqedR/7-crud-operations-for-routes)
